### DISCUSSION TITLE
About Platform Selection

### DISCUSSION THREAD

**Discussion by trillebart1** [hmes98318/r6-discord-bot#6] [Q&A] [answered]
Hello,

I want to use your bot without selection platform but I couldn't do that. I just want it to work on pc like -r6 [name]. Could you make it.

**Comment by hmes98318** [comment, marked as answer]
> Hello,
> 
> I want to use your bot without selection platform but I couldn't do that. I just want it to work on pc like -r6 [name]. Could you make it.

Hi,
I add the function of the default platform, you can check it at [here](https://github.com/hmes98318/R6Bot/releases).
